google-cloud-bigquery/test/google/cloud/bigquery/dataset_insert_test.rb

```diff
@@ -240,7 +240,7 @@
     }
     inserted_row_hash = {
       insertId: insert_id,
-      json: {"id"=>2, "name"=>"Gandalf", "age"=>1000, "weight"=>198.6, "is_magic"=>true, "scores"=>[100.0, 99.0, 0.001], "spells"=>[{"name"=>"Skydragon", "discovered_by"=>"Firebreather", "properties"=>[{"name"=>"Flying", "power"=>1.0}, {"name"=>"Creature", "power"=>1.0}, {"name"=>"Explodey", "power"=>11.0}], "icon"=>"eyJuYW1lIjoibWlrZSIsImJyZWVkIjoidGhlY2F0a2luZCIsImlkIjoxLCJkb2IiOjE1Mjc4ODI2NTV9CnsibmFtZSI6ImNocmlzIiwiYnJlZWQiOiJnb2xkZW5yZXRyaWV2ZXI/IiwiaWQiOjIsImRvYiI6MTUyNzk2OTA1NX0KeyJuYW1lIjoiamoiLCJicmVlZCI6Imlka2FueWNhdGJyZWVkcyIsImlkIjozLCJkb2IiOjE1MjgwNTU0NTV9Cg==", "last_used"=>"2015-10-31 23:59:56.000000+00:00"}], "tea_time"=>"15:00:00", "next_vacation"=>"2666-06-06", "favorite_time"=>"2001-12-19 23:59:59.000000"}
+      json: {"id"=>2, "name"=>"Gandalf", "age"=>1000, "weight"=>198.6, "is_magic"=>true, "scores"=>[100.0, 99.0, 0.001], "spells"=>[{"name"=>"Skydragon", "discovered_by"=>"Firebreather", "properties"=>[{"name"=>"Flying", "power"=>1.0}, {"name"=>"Creature", "power"=>1.0}, {"name"=>"Explodey", "power"=>11.0}], "icon"=> Google::Cloud::Bigquery::Convert.to_json_value(File.open("acceptance/data/kitten-test-data.json", "rb")), "last_used"=>"2015-10-31 23:59:56.000000+00:00"}], "tea_time"=>"15:00:00", "next_vacation"=>"2666-06-06", "favorite_time"=>"2001-12-19 23:59:59.000000"}
```

Comment:
I believe this has passed on Windows previously. Do you know why it is not matching now?

Reply:
I don't. The icon was slightly different from the expected. 🤷‍♂️

Reply:
Ruby will read text files on Windows differently than on POSIX, but the `b` in the read mode should suppress this.

Are the files in the `acceptance/data` directory being modified or changed somehow in the current Windows setup?

Reply:
The only place I know of that touches the acceptance data is https://github.com/googleapis/google-cloud-ruby/blob/0a2b7112593e5520db19ab3ba5588fc286f211ef/Rakefile#L632 . The files are identical on the windows machine I'm using to test currently.